pom.xml

```diff
@@ -18,145 +18,239 @@ specific language governing permissions and limitations
-						</goals>
-					</execution>
-				</executions>
-				<configuration>
-					<mainClass>org.apache.flink.tools.ci.licensecheck.LicenseChecker</mainClass>
-					<includePluginDependencies>true</includePluginDependencies>
-					<includeProjectDependencies>false</includeProjectDependencies>
-				</configuration>
-				<dependencies>
-					<dependency>
-						<groupId>org.apache.flink</groupId>
-						<artifactId>flink-ci-tools</artifactId>
-						<version>1.16-SNAPSHOT</version>
-					</dependency>
-				</dependencies>
-			</plugin>
-
-			<plugin>
-				<!-- Override directory for parent project -->
-				<groupId>org.commonjava.maven.plugins</groupId>
-				<artifactId>directory-maven-plugin</artifactId>
-				<executions>
-					<execution>
-						<id>directories</id>
-						<goals>
-							<goal>directory-of</goal>
-						</goals>
-						<configuration>
-							<property>rootDir</property>
-							<project>
-								<groupId>org.apache.flink</groupId>
-								<artifactId>flink-connector-aws</artifactId>
-							</project>
-						</configuration>
-					</execution>
-				</executions>
-			</plugin>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-enforcer-plugin</artifactId>
-				<executions>
-					<execution>
-						<id>dependency-convergence</id>
-						<goals>
-							<goal>enforce</goal>
-						</goals>
-						<configuration>
-							<skip>false</skip>
-						</configuration>
-					</execution>
-				</executions>
-			</plugin>
-		</plugins>
-	</build>
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.github.zentol.flink</groupId>
+        <artifactId>flink-connector-parent</artifactId>
+        <version>1.0</version>
+    </parent>
+
+    <groupId>org.apache.flink</groupId>
+    <artifactId>flink-connector-aws-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+
+    <name>Flink : Connectors : AWS : Parent</name>
+    <packaging>pom</packaging>
+    <url>https://flink.apache.org</url>
+    <inceptionYear>2022</inceptionYear>
+
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <scm>
+        <url>https://github.com/apache/flink-connector-aws</url>
+        <connection>git@github.com:apache/flink-connector-aws.git</connection>
+        <developerConnection>scm:git:https://gitbox.apache.org/repos/asf/flink-connector-aws.git</developerConnection>
+    </scm>
+
+    <properties>
+        <aws.sdk.version>2.17.247</aws.sdk.version>
+        <netty.version>4.1.77.Final</netty.version>
+        <flink.version>1.16.0</flink.version>
+        <flink.shaded.version>15.0</flink.shaded.version>
```

Comment:
Q. Why is this 15.0? We see this in the outputs

```
Warning:  Could not find artifact org.apache.flink:flink-sql-connector-dynamodb:jar:1.15.0 in central (https://repo.maven.apache.org/maven2)
Warning:  Could not resolve dependency with descriptor 'org.apache.flink:flink-sql-connector-dynamodb:1.15.0'.
Warning:  Please provide at least one resolvable old version using one of the configuration elements <oldVersion/> or <oldVersions/>.

```

Reply:
I copied this from elasticsearch actually. But I do see a 16.0 available. https://mvnrepository.com/artifact/org.apache.flink/flink-shaded-force-shading

Not sure on the warnings, will take a look.

Reply:
These things are unrelated. I have upgraded Flink shaded to 16.0. The warnings were from the `japicmp` plugin which is trying to compare binary compatibility. Given that this is the first version of the DDB connector there is no baseline to compare against. I have removed this plugin, we can add it back on the next version